CMakeLists.txt

```diff
@@ -499,6 +499,16 @@ include(CMakePushCheckState)
 
 print_versions()
 
+include(SwiftSharedCMakeConfig)
+
+# Support building Swift as a standalone project, using LLVM as an
+# external library.
+if(SWIFT_BUILT_STANDALONE)
+  swift_common_standalone_build_config(SWIFT)
+else()
+  swift_common_unified_build_config(SWIFT)
+endif()
```

Comment:
Might be nice to have a comment to explain that this needs to occur early to get the LLVM CMake macros (so someone doesn't try to move this around).